packages/firestore/scripts/run-tests.ts

```diff
@@ -0,0 +1,74 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as yargs from 'yargs';
+import { resolve } from 'path';
+import { spawn } from 'child-process-promise';
+
+const argv = yargs.options({
+  main: {
+    type: 'string',
+    demandOption: true
+  },
+  emulator: {
+    type: 'boolean'
+  },
+  persistence: {
+    type: 'boolean'
+  }
+}).argv;
+
+const nyc = resolve(__dirname, '../../../node_modules/.bin/nyc');
+const mocha = resolve(__dirname, '../../../node_modules/.bin/mocha');
+
+const env: { [key: string]: string } = {
```

Comment:
This seems way over complicated. `spawn` defaults to using the current environment, so just changing the environment of this process prepares the environment for the child.

```js
const env = process.env;
env.TS_NODE_CACHE = 'NO';
env.TS_NODE_COMPILER_OPTIONS = '{"module":"commonjs"}';
```

If that's not possible, it seems like it's worth a comment here.

Reply:
That works as is. Thanks for the simplification.